packages/time-series/lib/commands/ADD.ts

```diff
@@ -11,12 +11,18 @@ import {
   Timestamp
 } from '.';
 
+export interface TsIgnoreOptions {
+  MAX_TIME_DIFF: number;
+  MAX_VAL_DIFF: number;
```

Comment:
Is "IGNORE no values" supported? (https://github.com/RedisTimeSeries/RedisTimeSeries/pull/1543)
If it is - both `MAX_TIME_DIFF` and `MAX_VAL_DIFF` should be optional..

Reply:
doesn't seem like it, from code

```
int parseIgnoreArgs(RedisModuleCtx *ctx,
                    RedisModuleString **argv,
                    int argc,
                    long long int *ignoreMaxTimeDiff,
                    double *ignoreMaxValDiff) {
    int idx = RMUtil_ArgIndex("IGNORE", argv, argc);
    if (idx > 0) {
        long long maxTimeDiff;
        double maxValDiff;

        if (idx + 2 >= argc ||
            RMUtil_ParseArgs(argv, argc, idx + 1, "ld", &maxTimeDiff, &maxValDiff) !=
                REDISMODULE_OK) {
            RTS_ReplyGeneralError(ctx, "TSDB: Couldn't parse IGNORE");
            return TSDB_ERROR;
        }

        if (maxTimeDiff < 0 || maxValDiff < 0) {
            RTS_ReplyGeneralError(ctx, "TSDB: IGNORE arguments cannot be negative");
            return TSDB_ERROR;
        }

        *ignoreMaxTimeDiff = maxTimeDiff;
        *ignoreMaxValDiff = maxValDiff;
    }

    return TSDB_OK;
}
```

Reply:
So if not, why `MAX_TIME_DIFF` and `MAX_VAL_DIFF` are optional? also, rename them to `maxTimeDiff` and `maxValDiff` to match the command interface.

Reply:
if we want to require the user to specify them as 0, I'm good with that as well

Reply:
If the server does not have defaults we shouldn't as well..